pygmt/helpers/utils.py

```diff
@@ -172,6 +172,9 @@ def is_nonstr_iter(value):
     True
     >>> is_nonstr_iter((1, 2, 3))
     True
+    >>> import numpy as np
+    >>> is_nonstr_iter(np.array([1.0, 2.0, 3.0]))
+    True
```

Comment:
Do you want to test with `np.array(["abc", "def", "ghi"])`? :laughing:

By the way, what's the reason for adding this test?

Reply:
> Do you want to test with `np.array(["abc", "def", "ghi"])`?

I think yes. It should be `True`.

> By the way, what's the reason for adding this test?

Trying to implement the feature of varying transparency level (#615). `transparency` can be int, float or list/array.

I need to check if `transparency` is an list/array. If yes, then append it to the `extra_arrays`, which will be passed via `GMT_Put_Vector`.

Reply:
Ah I see, looks ok then.